modules/custom-post-types/portfolios.php

```diff
@@ -173,23 +160,17 @@ function setting_html() {
 				<a target="_blank" href="http://en.support.wordpress.com/portfolios/"><?php esc_html_e( 'Learn More', 'jetpack' ); ?></a>
```

Comment:
I think it's fine to have separate support pages.  I've drafted up a <a href="https://en.support.wordpress.com/?page_id=97757&preview=true">testimonials support page</a> that still needs screenshots added and to be reviewed.